test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/https/WorkbenchHttpsIntegrationTest.java

```diff
@@ -180,10 +189,10 @@ private SSLConnectionSocketFactory getSSLConnectionSocketFactory() {
             builder.loadTrustMaterial(null, new TrustAllStrategy());
             sslsf = new SSLConnectionSocketFactory(
                     builder.build(), SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
+
+            return sslsf;
         } catch (Exception e) {
-            Assertions.fail("Error in SSL setup", e);
+            throw new RuntimeException("Error in SSL setup", e);
```

Comment:
In general it returns RuntimeException when there is a problem with some test setup or something which is not related to system under test. If there is a problem with system under test then it should fail on asserts.

In this case the problem is in setup of SSL in HTTP client. It is not related to system under test.